kornia/augmentation/augmentation.py

```diff
@@ -573,7 +656,13 @@ def __init__(self, size: Union[int, Tuple[int, int]], align_corners: bool = True
         # Since PyTorch does not support ragged tensor. So cropping function happens batch-wisely.
         super(CenterCrop, self).__init__(p=1., return_transform=return_transform, same_on_batch=True, p_batch=p,
                                          keepdim=keepdim)
-        self.size = size
+        if isinstance(size, tuple):
```

Comment:
Can we provide an alternative here to remove Union type?

Reply:
If I remember right, the "overload" method worked for args Union types. I do not think there is a better way to replace the Union type in args.